kafka-ui-contract/src/main/resources/swagger/kafka-ui-api.yaml

```diff
@@ -1251,6 +1251,34 @@ paths:
         404:
           description: Not found
 
+  /api/streamtopology:
```

Comment:
We should do it on cluster level and per application

Reply:
done